src/css/components/grid.scss

```diff
@@ -32,3 +32,77 @@
     }
   }
 }
+
+.col,
+.row {
+  position: relative;
+}
+
+.row {
+  align-items: center;
+  flex-wrap: wrap;
+
+  // hack for react containing div requirement
+  // targets unclassed divs
+  > div:not([class]) {
+    align-items: center;
+    display: flex;
+    flex: 1 0 0;
+  }
+
+  &.row-start {
+    > div:not([class]) {
+      align-items: flex-start;
+    }
+  }
+
+  &.row-end {
+    > div:not([class]) {
+      align-items: flex-end;
+    }
+  }
+}
+
+.col-flex-1 {
+  flex: 1 0 0;
+}
+
+.col-flex-2 {
+  flex: 2 0 0;
+}
+
+.col-flex-none,
+.col-flex-0 {
+  flex: none;
+}
+
+$gutter: $grid-4;
+$gutter-offset: $grid-2; // $gutter / 2
```

Comment:
I haven't had good luck getting math operators to work well in the sass — keep throwing errors — so I hardcoded